tests/Unit/CrudPanel/CrudPanelTabsTest.php

```diff
@@ -7,6 +7,9 @@
 
 class CrudPanelTabsTest extends BaseDBCrudPanelTest
 {
+    private $horizontalTabsType = 'horizontal';
+    private $verticalTabsType = 'vertical';
```

Comment:
What happens for tab types that are just gibberish (e.g. a `fodsklfjdklfjdsl` tab type, or more likely a `vretical` type)?

Reply:
I would assume it was done this way to allow users to give them custom property names and display them however they choose when override the view files. Like, displaying them vertically, but on the right side instead of the left side when the user sets the tabs type to 'vertical_right'.

Reply:
I will try to cover this case when I have some time to get started on the integration tests...